source/technical/shares.rst

```diff
@@ -0,0 +1,234 @@
+.. _shares:
+
+====================
+Shares
+====================
+
+Chameleon provides a shared file system service through the `OpenStack Manila <https://docs.openstack.org/manila/latest/>`_ interface.
+With the service, you can create a shared file system, mount to the bare metal instances, and manage some of its properties, such as visibility.
+
+  .. hint::
+  
+    Chameleon shared file system service is currently backed by a CephFS. Same as our :ref:`object store <object-store>` service, the data is
+    replicated and the replication should provide good availability in case of hardware failures.
+
+    *Difference between shared file system and object store*
+	You can choose either shared file system or object store to store, manage, and share your data with your collaborators. The object store
+	is suitable for storing large objects at scale, but isn’t suitable for transactional data, as objects are immutable and updated in their
+	entirety. On the other hand, Chameleon shared file system maps the CephFS using file-based NFS protocol to the bare metal instances, and
+	the NFS protocol manages locking and data integrity processes required to provide multiple concurrent access to data. 
+
+The shared file system service is available at `CHI@UC <https://chi.uc.chameleoncloud.org/>`_ and `CHI@TACC <https://chi.tacc.chameleoncloud.org/>`_.
+Each region has its own service and the shares created at one region are not available to the other. As all other Chameleon services, you can create
+and manage your shares using both GUI and CLI.
+
+.. _storage_vlans:
+
+Storage VLANs
+================================
+
+To provide isolation among shares created by different projects, accessing a share requires the storage VLANs, which are special VLANs you can
+reserve to use. When reserving the storage VLANs, add `usage_type=storage` to the resource properties. To learn more about reserving VLANs, read
+the :ref:`reservations documentation <reservations>`. All bare metal instances that are created on the storage VLAN have access to all the project
+shares.
```

Comment:
```suggestion
.. _storage_network:

Storage Networks
================================

To provide isolation among shares created by different projects, accessing a share requires a storage network, which are special networks you can
reserve to use. When reserving a storage network, add `usage_type=storage` to the resource properties. To learn more about reserving networks, read
the :ref:`reservations documentation <reservations>`. All bare metal instances that are created on the storage network have access to all the project
shares.
```

I think we refer to reservable networks as networks and not VLANs, I think just saying "network" gets the point across and is less technical for, e.g., storage researchers who many not know what a VLAN is or how it relates.